src/app/issuer/models/issuer.model.ts

```diff
@@ -122,19 +122,28 @@ export class Issuer extends ManagedEntity<ApiIssuer, IssuerRef> {
 		return this.update();
 	}
 
-    /**
-     * Evaluates if the current user can create badges.
-     * This is the case if all of the following conditions are fulfilled:
-     * - the issuer is verified
-     * - there is a logged in user
-     * - the logged in user has either the owner or editor role for this issuer
-     *
-     * @returns {boolean}
-     */
-    get canCreateBadge(): boolean {
-        return this.apiModel.verified &&
-            (this.currentUserStaffMember?.canEdit ?? false);
-    }
+	/**
+	 * Evaluates if the current user can edit issuer.
+	 * This is only if the user is an owner.
+	 *
+	 * @returns {boolean}
+	 */
+	get canEditIssuer(): boolean {
```

Comment:
Can this function be removed? There seem to be two ```canEditIssuer``` functions in this file, i think we should use this one: 
``` 
get canEditIssuer(): boolean {
		return this.isOwner;
	}
```

Reply:
> Can this function be removed? There seem to be two `canEditIssuer` functions in this file, i think we should use this one:
> 
> ```
> get canEditIssuer(): boolean {
> 		return this.isOwner;
> 	}
> ```

We need both functions. I just updated its name.
Like in `canEditBadge` and `canCreateBadge`, 1st fun. to specify the roles and the 2nd to be called.